src/Game.elm

```diff
@@ -46,6 +46,8 @@ type ScreenCoordinates
 
 type alias Model =
     { world : World Data
+    , ballTextures : Dict Int (Material.Texture Color)
+    , roughnessTexture : Material.Texture Float
```

Comment:
Reinitializing required storing the ball and roughness textures in the game model.